### PR TITLE
docs: added `missingNgForOfLet` extended diagnostics details page

### PR DESCRIPTION
**Pull request by AndrewKushnir**
## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by jessicajaniuk — APPROVED** [review]
LGTM

reviewed-for: public-api

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: public-api

File: aio/content/extended-diagnostics/NG8105.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  template: `&lt;div *ngFor="item of items"&gt;{{ item }}&lt;/div&gt;`,
+  // &hellip;
+})
+class MyComponent {
+  items = [1, 2, 3];
+}
+
+</code-example>
+
+## How to resolve the problem?
```
Comment (bob-watson) [inline aio/content/extended-diagnostics/NG8105.md:22]:
```suggestion
## How to resolve the problem
```
Generally, technical documentation is written as though it's speaking to the reader. In that context, using questions in the sub headings sounds odd.

File: aio/content/extended-diagnostics/NG8105.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+The fix is to add a missing `let` keyword.
```
Comment (bob-watson) [inline aio/content/extended-diagnostics/NG8105.md:24]: As an instruction to fix the problem, the imperative voice tends to be clearer.

```suggestion
Add the missing `let` keyword.
```
There's no need to prefix the action with the context as the context is given by the preceding heading.

**Review by bob-watson — CHANGES_REQUESTED** [review]
Just a couple of comments.

**Comment by AndrewKushnir** [comment]
@bob-watson thanks for the review, I've addressed the feedback. Could you please re-review? Thank you.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 7b7d5e1199e81da4ecaab1f51767f6f38317716a.

**Comment by AndrewKushnir** [comment]
@bob-watson FYI I've merged this PR and can create a followup one if you have any feedback.

**Pull request closed without merging**